Assets/Prefabs/Moveable/UBlock.prefab

```diff
@@ -132,6 +121,45 @@ MonoBehaviour:
   sizeMax: {x: 5, y: 2, z: 20}
   canRandomizeColor: 0
   ratioSize: {x: 0.3, y: 1, z: 0.1}
-  sizeAdjustement: 0.999
+  sizeAdjustment: 0.999
   textureUVOverride: 0
   typicalOrigin: 1
+--- !u!65 &4099879433487610999
```

Comment:
So nothing was modified/changed that would alter the overall functionality of the object(s). Kozzy spotted and mentioned to me that the agent can go through the edges of some of the movable objects (L,U,J-Blocks). This was because the incorrect collision setup was implemented originally as the colliders were not optimised to cover the entire object's physical mesh, resulting in gaps where the agent could go through. The changes would have no impact on the olympics suite (if any, it would make everything better as now these objects have proper collision setups) :)

Let me know if you have any other questions :)